cpp/include/Ice/Communicator.h

```diff
@@ -20,41 +20,37 @@ namespace Ice
     class LocatorPrx;
     class RouterPrx;
 
-    /// The central object in Ice. One or more communicators can be instantiated for an Ice application.
-    /// @see Logger
-    /// @see ObjectAdapter
```

Comment:
I don't see the point of these random `@see`. In particular, why add a `@see` to types in function signatures?